sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -3659,6 +3663,23 @@ SIREPO.app.directive('moderationRequest', function(appState, errorService, panel
                     }
                 );
             };
+
+            function reasonOk(reason) {
+                if (reason && reason.length > 4) {
```

Comment:
`return reason && reason.length > 4` except that you need to strip reason, because it might contain blanks on either end.

Reply:
Good point